features-json/css-appearance.json

```diff
@@ -261,7 +261,8 @@
   "notes":"",
   "notes_by_num":{
     "1":"The appearance property is supported with the `none` value, but not `auto`. Webkit, Blink, and Gecko browsers also support additional vendor specific values.",
-    "2":"Microsoft Edge and IE Mobile support this property with the `-webkit-` prefix, rather than `-ms-` for interop reasons."
+    "2":"Microsoft Edge and IE Mobile support this property with the `-webkit-` prefix, rather than `-ms-` for interop reasons.",
+    "3":"-moz-appearance:none on a combobox not remove the dropdown arrow"
```

Comment:
HTML doesn't have comboboxes. Do you mean `<select>`?
Also, this is missing a verb; should be "does not".